tests/components/paperless_ngx/snapshots/test_sensor.ambr

```diff
@@ -0,0 +1,301 @@
+# serializer version: 1
+# name: test_sensor_platfom[sensor.192_168_69_16_correspondents-entry]
```

Comment:
Another reason to not have the host as device name is that you dont get the IP as entity_id